qt/widgets/common/test/ConvolutionFunctionModelTest.h

```diff
@@ -361,11 +361,11 @@ class ConvolutionFunctionModelTest : public CxxTest::TestSuite {
     TS_ASSERT_EQUALS(
         fitFunctionAsString,
         "composite=MultiDomainFunction,NumDeriv=true;(composite=Convolution,"
-        "FixResolution=true,NumDeriv=true,$domains=i;name=Resolution,Workspace="
+        "NumDeriv=true,FixResolution=true,$domains=i;name=Resolution,Workspace="
```

Comment:
If a string existed from the current implementation I assume the order here wouldn't matter and the generator can handle it?

Reply:
As far as I can tell it’s able to handle the new order. The ConvolutionFunctionModel is used in the ConvFunctionModel class which is subsequently used in the Indirect data analysis GUI and the tests for ConvFunctionModel and interface seem to behave as normal.